exporters/keystone.go

```diff
@@ -69,6 +72,11 @@ func ListProjects(exporter *BaseOpenStackExporter, ch chan<- prometheus.Metric)
 
 	ch <- prometheus.MustNewConstMetric(exporter.Metrics["projects"].Metric,
 		prometheus.GaugeValue, float64(len(allProjects)))
+	for _, p := range allProjects {
+		ch <- prometheus.MustNewConstMetric(exporter.Metrics["project_info"].Metric,
+			prometheus.GaugeValue, 1.0, strconv.FormatBool(p.IsDomain),
```

Comment:
Why not setting the metric value to p.Enabled or not (1, or 0) rather than a static 0?

Reply:
I exposed a `1` here as I consider this metric a "pseudo-metric" hence the suffix `_info`. I chose this value based on the prometheus best practices documentation (https://prometheus.io/docs/practices/naming/#metric-names), which points to this article suggesting a value of 1 for pseudo-metrics (https://www.robustperception.io/exposing-the-software-version-to-prometheus).

This is rather useful because you can then use this pseudo metric to add labels to a given query by taking advantage of the `*` operator.

Would this be ok with you?

Reply:
Thinking about it, it might be a good idea to have an other metric like `openstack_identity_project_enabled` which we could set to either `0` or `1` and keep this info out of the `project_info` metric? As I'm rather new to prometheus let me know if this is a terrible idea :)

Reply:
OK, I think I agree with the idea of leaving this one as a pseudo-metric.